doc/help/MV.md

```diff
@@ -0,0 +1,23 @@
+kdb-mv(1) -- Move keys within the key database
+==============================================
+
+## DESCRIPTION
+
+This command moves key(s) in the Key database.  
+You can move keys to another directory within the database or even below another key.  
+
+## USAGE
+
+`kdb mv <source> <dest>`  
+
+Where `source` is the path of the key(s) you want to copy and `dest` is the path where you would like to move the key(s) to.  
+
+## EXAMPLES
+
+To move a KeySet:  
```

Comment:
I would simply use "To move many keys at once"

But you should specify which keys are included and which not. (But not in the example, but already in USAGE)